app/models/badges/ewok.rb

```diff
@@ -0,0 +1,19 @@
+class Badge::Ewok
+  BADGE_ID = 5
+  DISPLAY_NAME = 'Ewok'
+  SLUG = 'ewok'
+
+  include Badge
+
+  def description
+    'Given to users who have attended 5 lessons.'
+  end
+
+  def notification_bonus_message
+    'Congratulations! You are on the path.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.